tutormfe/templates/mfe/build/mfe/env.config.jsx

```diff
@@ -0,0 +1,48 @@
+{{- patch("mfe-env-config-static-imports") }}
+
+{{- patch("mfe-env-config-head") }}
```

Comment:
Having two patch statements in quick succession is a clue that we should remove one. If some patches need to be at the bottom, then they can be added with a lower priority.

Reply:
OK let's keep those patches then.